src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -899,41 +899,39 @@ Sentry owner or manager permissions are required to install this integration. Sl
 
-1. After you've connected to the correct workspace, click **Continue**.
+1. You can toggle the Slack workspace you want to connect in the dropdown menu in the upper right corner of the authentication window. To set up multiple workspaces, go through this process for each separate workspace. Once you've chosen the correct workspace, click **Allow**.
 
-1. On the next page, you can choose the scope of channels you’d like Sentry to access. You can specify all public channels or choose specific channels. Then click **Install**.
+    [{% asset slack-allow-install.png %}]({% asset slack-allow-install.png @path %})
 
-    [{% asset slack-auth-channel-pref.png %}]({% asset slack-auth-channel-pref.png @path %})
+1. Your integration details page will refresh and show the Slack workspace you just added.
 
-1. Your Organization Integrations page will refresh and show the Slack workspace you just added.
+    [{% asset slack-install-success.png %}]({% asset slack-install-success.png @path %})
 
-    [{% asset slack-add-workspace-success.png %}]({% asset slack-add-workspace-success.png @path %})
-
-    You’ll also see the Sentry app join the Slack channels you specified right away.
-
-    [{% asset slack-sentry-integration-in-channel.png %}]({% asset slack-sentry-integration-in-channel.png @path %})
-
-Now your Slack integration is available to all projects in your Sentry organization. See the next section to configure your notification settings.
+Now your Slack integration is available to all projects in your Sentry organization. In order to get Slack notifications for private channels, please make sure to add the Slack app to the channel. You can do so easily by using `@sentry` to invite the bot to the channel. See the next section to configure your notification settings.
 
 #### Configure Alert Rules
 
-1. Confirm your Slack workspace is configured globally for your Sentry organization by navigating to Organization Settings > **Integrations**.
+1. Confirm your Slack workspace is configured globally for your Sentry organization by navigating to Organization Settings > **Integrations**. Click on **Configuration(s)** to go to the configuration details page. 
+
+    [{% asset slack-directory-installed.png %}]({% asset slack-directory-installed.png @path %})
+    
+1. Here you will see all the Slack workspaces connected to Sentry. Click on **Configure** for the workspace you just added.
 
-    [{% asset slack-add-workspace-success.png %}]({% asset slack-add-workspace-success.png @path %})
+    [{% asset slack-install-success.png %}]({% asset slack-install-success.png @path %})
 
-1. Click **Configure** and you’ll see the available Slack workspaces for your project. From this page, you can click on **Add Alert Rule** for this project to go directly to your Alert Rule settings.
+1. From this page, you can click on **Add Alert Rule** for this project to go directly to your Alert Rule settings.
```

Comment:
Suggest, "Click **Add Alert Rule** for this project to navigate to Alert Rule settings."